echo-terminal.md

```diff
@@ -0,0 +1,35 @@
+# Echo Terminal Snippet Web Component
+
+The Echo Terminal Snippet can be loaded as a web component via JSDelivr CDN.
+
+The JSDelivr URLs below point to the latest commit in the main branch, ensuring changes to the snippet are automatically deployed to consuming websites.
+
+[See here for more options when testing using JSDelivr](https://www.jsdelivr.com/).
+
+## Usage
+
+The Echo terminal can be loaded via both JS and HTML.
+
+### Loading with Javascript (preferred)
+Use this method if you have access to custom JS.
+
+```javascript
+// Custom JS injection
+!function(e){if(!document.getElementById(e)){var t=document.createElement("script");t.src="https://cdn.jsdelivr.net/gh/novuhq/docs/echo-terminal.min.js",t.type="text/javascript",t.crossOrigin="anonymous",t.id=e;var n=document.getElementsByTagName("script")[0];n?n.parentNode.insertBefore(t,n):document.body.appendChild(t)}}("nv-echo-terminal-loader");
```

Comment:
#til jsdelivr added GitHub source code delivery 👏 👏 👏

Reply:
Nit-pick: Do we need a cache buster in the url in case we deploy a new `echo-terminal.js`? I guess we can also use a relative path via Mintlify.

Reply:
Yes we do, good catch. I'll add a cache invalidation GH action to trigger whenever this file changes on `main`.

Reply:
Added cache invalidation: https://github.com/novuhq/docs/pull/558/files#diff-9836448ac7f7de6b0fa1069ee830870c1496963ee3255b86fd729962b665580f